### DISCUSSION TITLE
Can you use medialibrary (free version) for Vue instead of Blade?

### DISCUSSION THREAD

**Discussion by fickezapad** [spatie/laravel-medialibrary#2824] [Q&A] [answered]
I saw that the PRO version offers: vue,react and blade components but im not sure about the free version? Can you also upload video and audio files besides images?

**Comment by erikn69** [comment, marked as answer]
>Can you use medialibrary (free version) for Vue instead of Blade? 

Yes, you can

>Can you also upload video and audio files besides images?

Yes

**Reply by fickezapad** [reply]
Thanks!

**Reply by fickezapad** [reply]
What about this [answer](https://github.com/spatie/laravel-medialibrary/issues/602) which says you cannot access the library in vuejs?

**Reply by erikn69** [reply]
>The package does not provide a vue.js integration, it's a bit out of scope for this package.

it means you have to do your own front end implementation, the package only offers backend
